content/integrations/AWS-Integration/AWS-IAM-Installation.md

```diff
@@ -10,53 +10,63 @@ weight: 2
+Setting up an AWS integration via IAM Role is a three step process:
+
+1. Create a new AWS integration in Metricly.
+ - Optionally, define data filters for AWS elements to be included/excluded in Metricly using tags (key-value pair).
+2. Create an IAM role in your AWS Console.
+3. Add your IAM Role's ARN to your AWS integration in Metricly.
 
-Create a new AWS integration in Metricly using an IAM read-only role.
-Optionally, filter your AWS elements for inclusion in Metricly by creating or choosing an existing tag (key-value pair), then assigning that tag to the desired elements in AWS.
 
 {{% notice tip %}}
 If you already have an existing IAM role for Metricly but it does not include a policy for Cost Explorer, skip to the last section.
 {{% /notice %}}
 
-### Step 1: Create a new AWS integration
-1. From the top navigation menu, select Integrations.
-2. Click the Amazon Web Services card.
-3. Type a name for the new AWS integration. Ensure that Data Collection is selected.
-4. For AWS Authentication, select IAM Role.
+### Step 1: Create a new AWS integration in Metricly
+1. Login to Metricly and select the **Integrations** icon.
+![integrations-icon](/images/AWS-IAM-Installation/integrations-icon.png)
+2. Select the **Amazon Web Services** card.
+3. Provide a name for the new AWS integration.
+4. Enable **Detailed Billing** and **Explorer API**.
+   - Once you have finished all setup on this page, see the [Detailed Billing steps](/integrations/aws-integration/aws-detailed-billing).
+4. For Authentication, select **IAM Role**.
 5. In a separate, new tab, open your AWS console.
 
-### Step 2a: Create Read Only Role (with standard permissions)
-1. Log in to your AWS Identity & Access Management (IAM) Console.
-2. Once in the IAM dashboard, navigate to the Roles section.
-3. Click Create New Role.
-4. For Role Name, type Metricly and click Next Step.
-5. For Role Type, select Role for Cross-Account Access.
-6. Click Select next to the “Provide access between your AWS account and a 3rd party AWS account” option.
-7. On the tab that has the AWS Integration Setup page open, copy the Account ID and the External ID provided to you.
-8. On the tab that has the AWS console open, paste your Account ID and External ID into the appropriate fields. Leave Require MFA unchecked. Click Next Step.
-9. For Attach Policy, add all of the following:
+### Step 2: Create Read Only Role (with standard permissions)
+
+1. Log in to your **AWS Console**.
+2. In **Find Services**, search for `IAM` and select the result.
+![select-IAM](/images/AWS-IAM-Installation/select-iam.png)
+3. Select **Roles**.
+4. Select **Create role**.
+5. Select **Another AWS Account**
+![another-aws-accnt](/images/AWS-IAM-Installation/another-aws-accnt.png)
+6. Provide the **Account ID** from your Metricly AWS integration. _Leave Require MFA unchecked_.
+7. Select **Next: Permissions**.
+8. For Attach permission policies, add all of the following:
  - CostExplorerAPIAccess
```

Comment:
This has to be created by the user first. See https://docs.metricly.com/integrations/aws-integration/aws-iam-installation/#add-inline-policy-to-existing-iam-role.